index.js

```diff
@@ -144,8 +148,16 @@ async.forEach(inputFiles, compile, onError);
 function fsWatcher(entryPoints) {
   var watchedFiles = entryPoints;
   var index = {}; // source files by entry point
+  var opts = {};
+
+  if (argv.poll || typeof argv.poll === 'number') {
+    opts.usePolling = true;
+    opts.interval = argv.poll !== true
+      ? argv.poll
+      : undefined;
```

Comment:
@watilde Should be:
```
  if (argv.poll) {
    opts.usePolling = true;
  }
  if (typeof argv.poll === 'number') {
    opts.interval = argv.poll;
  }
```
So that:

> Omitting the interval will default to 100ms.

See https://github.com/paulmillr/chokidar#performance.